examples/simple-jsbeautifyrc/blade/expected/test.blade.php

```diff
@@ -0,0 +1,32 @@
+<html>
+
+<head>
+  <title>App Name - @yield('title')</title>
+</head>
+
+<body>
+  @section('sidebar') This is the {{ $mater }} sidebar.
+  @show
+
+  <div class="container">
+    @yield('content')
+  </div>
+
+  @component('alert')
+  @slot('title') Forbidden
+  @endslot
+  <strong>Whoops!</strong> Something went wrong!
+  @endcomponent
+
+  @foreach ($users as $user)
+  @if ($user->type == 1)
+  @continue
+  @endif
+  <li>{{ $user->name }}</li>
+  @if ($user->number == 5)
+  @break
+  @endif
+  @endforeach
+</body>
+
+</html>
```

Comment:
There is a newline here on line 33, although GitHub doesn't show it very well.  Please remove it and AppVeyor should pass.  Travis will still fail due to an unrelated issue.

Reply:
Hmm... That's strange...
Other language tests seems also having newlines...

Reply:
Sorry, not for html... haha
I'll fix it :)

Reply:
It was a specific issue with these two example files.  The original has a newline, but when beautified that line space was taken up.